py-polars/polars/lazyframe/frame.py

```diff
@@ -4337,12 +4350,14 @@ def explode(
         └─────────┴─────────┘
 
         """
-        columns = parse_as_list_of_expressions(columns, *more_columns)
+        columns = parse_as_list_of_expressions(
+            *expand_selectors(self, columns, *more_columns)
+        )
         return self._from_pyldf(self._ldf.explode(columns))
 
     def unique(
         self,
-        subset: str | Sequence[str] | None = None,
+        subset: str | Sequence[str] | SelectorType | None = None,
```

Comment:
This also supports `Sequence[SelectorType]`:

```suggestion
        subset: str | Sequence[str] | SelectorType | Sequence[SelectorType] | None = None,
```